tutorials/nservicebus-101/lesson-1/tutorial.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "NServiceBus 101 Lesson 1: Hello world"
+component: Core
+---
+
+Every framework needs a Hello World, and for NServiceBus, this is it. There's no time to waste, so let's get started!
+
+
+## Objectives
+
+In this lesson we will set up a new development machine with NServiceBus.
+
+By the end of this lesson, you will have learned:
+
+* How to set up a machine for NServiceBus development
+* What a messaging endpoint is and how to create one
+
+
+## Prerequisites
+
+There are a few things we need in order to build a solution with the latest version of NServiceBus.
+
+* This course uses Visual Studio 2015 and .NET Framework 4.6.1, although NServiceBus only requires .NET Framework 4.5.2.
+* NServiceBus requires queuing infrastructure (a message transport) to move messages around. This course uses Microsoft Message Queuing (MSMQ) as a default.
+  * The [Particular Platform Installer](/platform/installer/) will install MSMQ and its dependencies for you. [Download and run the Platform Installer]() to get started, or [follow these instructions to configure MSMQ manually](/nservicebus/msmq/#nservicebus-configuration).
+  * If you don't have sufficient privileges to install MSMQ, you can [use the SQL Server transport as a substitute](../using-sql-transport.md). 
```

Comment:
this is not something the tutorials should be solving. all the samples use MSMQ by default. so drop the "SQL Server transport as a substitute"